common/ocp-config/component/template.yml

```diff
@@ -0,0 +1,195 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: component-template
+  app: "${PROJECT}-${COMPONENT}"
+parameters:
+- name: PROJECT
+  displayName: Application
+  description: The name of the application project.
+  required: true
+- name: COMPONENT
+  displayName: Component
+  description: The name of the application component.
+  required: true
+- name: ENV
+  displayName: Environment
+  description: The environment stage.
+  required: true
+- name: MEMORY_LIMIT
+  displayName: Memory Limit
+  description: Maximum amount of memory available for the container.
+  value: 256Mi
+- name: MEMORY_REQUEST
+  displayName: Memory Request
+  description: Minimum amount of memory requested for the container.
+  value: 128Mi
+- name: CPU_LIMIT
+  displayName: CPU Limit
+  description: Maximum CPU (milli) cores available for the container.
+  value: 100m
+- name: CPU_REQUEST
+  displayName: CPU Request
+  description: Minimum CPU (milli) cores requested for the container.
+  value: 50m
+- name: TAGVERSION
+  displayName: tag version
+  description: The version to be used.
+  value: latest
+  required: true
+- name: COMPLETION_DEADLINE_SECONDS
+  displayName: Completion Deadline Seconds
+  description: how long the docker build of the component can last before it is canceled
+  value: "1800"
+- name: READNINESS_PATH
```

Comment:
There is a type in all the readiness variables.

Reply:
Oh my :( Thanks for the catch! Fixed in 949fc12.